src/ucp/wireup/wireup_cm.c

```diff
@@ -485,7 +485,6 @@ ucp_cm_client_resolve_cb(void *user_data, const uct_cm_ep_resolve_args_t *args)
     ucp_wireup_ep_t *cm_wireup_ep;
     char addr_str[UCS_SOCKADDR_STRING_LEN];
 
-    UCS_ASYNC_BLOCK(&worker->async);
```

Comment:
assert is locked?

Reply:
we're checking it in `UCP_EP_CM_CALLBACK_ENTER` macro at the begging of the function

Reply:
nice.